src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/expense/ExpenseController.java

```diff
@@ -0,0 +1,55 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.expense;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.expense.ExpenseService;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.expense.ExpenseResponseDTO;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("expense")
+public class ExpenseController {
+
+    private final ExpenseService expenseService;
+
+    public ExpenseController(ExpenseService expenseService) {
+        this.expenseService = expenseService;
+    }
+
+    @PostMapping("create")
```

Comment:
Transformar rota para o padrão RESTful, pois quando irá ser criado irá só passar no método POST `api/expenses`
```suggestion
    @PostMapping
```